<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>40</y>
     <width>971</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Başlamak için bir PDF dosyası seçin...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_select_file">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lütfen dosyanızı seçiniz:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_select">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>80</y>
     <width>100</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dosya Seç</string>
   </property>
  </widget>
  <widget class="QLabel" name="main_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>751</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PDF Dönüştür</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_source">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>PDF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WORD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>JPG</string>
    </property>
   </item>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>-120</x>
     <y>160</y>
     <width>971</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_goingtoconvert">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dönüştürülecek dosya formatı</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_output">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>250</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>PDF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WORD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>JPG</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_tobeconverted">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>230</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dönüşecek dosya formatı</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_select_file_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_convertSave">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dönüştür ve Kaydet</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>390</y>
     <width>231</width>
     <height>111</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Bu bölüm güncelleme almaya devam edecektir. İlginiz için teşekkür ederim.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:14pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;mozgor19.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
